corehq/apps/accounting/tasks.py

```diff
@@ -1049,10 +1049,8 @@ def calculate_users_in_all_domains(today=None):
                 num_users=num_users,
                 record_date=record_date
             )
-        except IntegrityError:
-            pass
-        except Exception as e:
+        except _ as e:
```

Comment:
I haven't seen `except _` before, what does that do?

Reply:
I thought I remember reading somewhere that it will catch any errors for you, but A) I could be wrong and don't want to press my luck too much and B) I don't see it anywhere else in HQ (I guess we tend to use `except:` or `except Exception as e:`)
I'm going to switch this back to `except Exception as e:`